### ISSUE TITLE
Add AlephWeb (Marathon game series) to EmuOS

### ISSUE THREAD

**Issue by dorr1** [Emupedia/emupedia.github.io#56]
Add AlephWeb, a port of the Aleph One source port to Javascript.
Git repo: https://github.com/TotallyGatsby/AlephWeb
It is inactive, so it may not work, but if it does it would make a good addition.

**Comment by ZaDarkSide** [comment]
Yeah, it's not a complete port, I just tested and it seems to be only at the stage of rendering a map level.
